internal/provider/resource_user_test.go

```diff
@@ -0,0 +1,44 @@
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAcc_ResourceUser_Basic(t *testing.T) {
+	userName := acctest.RandomWithPrefix("tfuser")
```

Comment:
this generates something like `blahblah-tfuser` when parsing the id and name in the `resourceModelRead` using 
```
id := strings.Split(d.Id(), "-")
name := id[1]
```
it will fail. You probably need to use `strings.LastIndex` or get the latest token.

Reply:
Thanks for spotting this, yes you are correct! I changed the SetID to use a colon instead of a dash, I see that is used across other resources such as models and applications. I could still use the last index like you mentioned. Thoughts?